packages/services/f-feature-management/src/index.js

```diff
@@ -1,38 +1,10 @@
-const logValueRequest = (func, key, value) => {
-    console.log(`${func.name}: ${key} => ${value}`); // es-lint-disable
-};
-
-function getBooleanValue (key) {
-    logValueRequest(getBooleanValue, key, true);
-    return true; // TODO: implementation
-}
-
-
-function getIntegerValue (key) {
-    logValueRequest(getIntegerValue, key, 0);
-    return 0; // TODO: implementation
-}
-
-
-function getStringValue (key) {
-    logValueRequest(getStringValue, key, '');
-    return ''; // TODO: implementation
-}
-
-function getValue (key) {
-    logValueRequest(getValue, key, null);
-    return null; // TODO: implementation
-}
-
-export default function (scope) {
-    const fullKey = key => (scope
-        ? `${scope}::${key}`
-        : key);
-
-    return {
-        getBooleanValue: key => getBooleanValue(fullKey(key)),
-        getIntegerValue: key => getIntegerValue(fullKey(key)),
-        getStringValue: key => getStringValue(fullKey(key)),
-        getValue: key => getValue(fullKey(key))
-    };
+import createFeatureManagementInstance from './createFeatureManagementInstance';
+
+/**
+ * Returns an instance of Feature Management.
+ * @param {object} settings - json and contextGetter must be set
+ * @returns Feature Management object to allow querying of features.
+ */
+export default function (settings) {
+    return createFeatureManagementInstance(settings);
```

Comment:
Is it up to the consuming application how often this is updated? Assuming it'll end up being once per request rather than keeping it the same between users?

Reply:
What do you mean by "updated"?

Reply:
I guess calling `initFeatures`, if the features are updated

Reply:
That's something we'll need to think about in due course.  I expect we'll have a poller at some point.  But just sidestepping all those questions for this PR.